app.py

```diff
@@ -63,56 +67,80 @@ def about():
 def send_js(path):
     return send_from_directory('scripts', path)
 
+@app.route('/images/<path:path>')
+def send_img(path):
+    return send_from_directory('images', path)
+
 @app.route('/temp/<path:path>')
 def send_temp(path):
     return send_from_directory('temp', path)
 
-def delete_temp_contents():
-    files = glob.glob('temp/*')
-    for f in files:
-        os.remove(f)
-
 @app.route('/upload', methods=['GET', 'POST'])
 def upload():
     if request.method == 'POST':
+        # Save audio file from POST data
         f = request.files['audio_data']
         f.save('temp/upload.wav')

```

Comment:
Will the temp file name upload.wav conflict if two users upload or record concurrently? (maybe a bit optimistic of me to assume we get two concurrent users...)

Reply:
Yeah but I can't think of an easy fix so we can worry about it in the future